SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/ChannelGroup.cs

```diff
@@ -122,6 +123,14 @@ public void SetNetworkRequestAllowed(bool isAllowed)
             }
         }
 
+        public void CheckPendingLogs()
```

Comment:
let's mark this method as private

Reply:
by the way, this check will be called for each channel in the array after each adding an additional new channel?
can we call this method only once for each channel after adding it to the group?

Reply:
> let's mark this method as private

We can't mark it private, it is coming from interface `IChannel`.

> by the way, this check will be called for each channel in the array after each adding an additional new channel?
> can we call this method only once for each channel after adding it to the group?

No, `checkPendingLogs` will be invoked only on the channel which was added.